packages/api-headless-cms/src/content/plugins/models/contentModel.model.ts

```diff
@@ -212,10 +212,20 @@ export default ({ createBase, context }: { createBase: Function; context: CmsCon
                             if (field === "id") {
                                 continue;
                             }
-
                             if (!this.fields.find(item => item.fieldId === field)) {
                                 break indexesFor;
                             }
+
```

Comment:
1. Please put a space between `//` and `if`.
2. Make the word `if` => `If`.
3. Error message - change it to `Cannot create a new index with the "${item.fieldId}" field included because the field accepts multiple values.`